core/types/transaction_signing.go

```diff
@@ -254,6 +254,22 @@ func (sg Signer) SenderWithContext(context *secp256k1.Context, tx Transaction) (
 		// id, add 27 to become equivalent to unprotected Homestead signatures.
 		V.Add(&t.V, u256.Num27)
 		R, S = &t.R, &t.S
+	case *SignedBlobTx:
+		if !sg.dynamicfee {
+			return common.Address{}, fmt.Errorf("dynamicfee tx is not supported by signer %s", sg)
+		}
+		if t.GetChainID() == nil {
+			if !sg.chainID.IsZero() {
+				return common.Address{}, ErrInvalidChainId
+			}
+		} else if !t.GetChainID().Eq(&sg.chainID) {
+			return common.Address{}, ErrInvalidChainId
+		}
+		// ACL and DynamicFee txs are defined to use 0 and 1 as their recovery
+		// id, add 27 to become equivalent to unprotected Homestead signatures.
+		V.Add(t.Signature.GetV(), u256.Num27)
+		R, S = t.Signature.GetR(), t.Signature.GetS()
```

Comment:
You will also need to update the implementation of SigningHash SignedBlobTransactions as I notice it uses a prefixedRlpHash whereas the spec requires a prefixedSSZ hash.

Reply:
Updated